### ISSUE TITLE
LikedSongsNew doesnt show

### ISSUE THREAD

**Issue by joeth27** [JustSxm/SpotifyMigrator#1]
Im not really familiar with node js but after following the instructions and taking a brief look, the page stays on likedsongs.html after logging into the old account. When following the pages instructions, you end up adding all the liked songs to your old account again but if you log into the new account on the 2nd login, everything works just fine. (I guess it's because LikedSongsNew.html doesn't load.)

**Comment by joeth27** [comment]
Also to add to this, it messed up the order of my liked songs but its not a big deal.

**Comment by JustSxm** [comment]
> Im not really familiar with node js but after following the instructions and taking a brief look, the page stays on likedsongs.html after logging into the old account. When following the pages instructions, you end up adding all the liked songs to your old account again but if you log into the new account on the 2nd login, everything works just fine. (I guess it's because LikedSongsNew.html doesn't load.)

This is the intended behavior, first time is old and next time is new. Maybe I forgot to change the page. The difference between the two page is only "old" and "new" so I might also need to make it more visible.




> Also to add to this, it messed up the order of my liked songs but its not a big deal.

It messed them up a bit but keep the same order , right? this was to make it faster but i can slow it down so it be the same exact order

**Comment by joeth27** [comment]
> It messed them up a bit but keep the same order , right? this was to make it faster but i can slow it down so it be the same exact order

To be honest, I think this was because I accidentally logged in twice and also on the same account so maybe it was trying to transfer at the same time or I interrupted it perhaps.
>This is the intended behavior, first time is old and next time is new. Maybe I forgot to change the page. The difference between the two page is only "old" and "new" so I might also need to make it more visible.

Yeah when I used it, it showed as old both times but making it more visible can't hurt. I got it to work anyway just by logging in on my old account first and new account afterwards (despite the page being the same), so it's all good. Thank you for the amazing program, made my life a lot easier in transferring liked songs in the "proper" way especially compared to other services online.
